tests/remote_config/test_remote_configuration.py

```diff
@@ -282,6 +283,53 @@ def validate(data):
         interfaces.library.validate_remote_configuration(validator=validate)
 
 
+@released(cpp="?", dotnet="2.36.0", golang="?", java="?", php_appsec="?", python="?", ruby="?", nodejs="?")
+@coverage.basic
+@scenarios.remote_config_mocked_backend_asm_features
+class Test_RemoteConfigurationExtraServices:
+    """Tests that extra services are sent in the RC message"""
+
+    remote_config_is_sent = False
+
+    def setup_tracer_extra_services(self):
+        self.r_outgoing = weblog.get("/createextraservice/?serviceName=extraVegetables")
+
+    def test_tracer_extra_services(self):
+        """ test """
+
+        def remote_config_asm_extra_services_available(data):
+            if data["path"] == "/v0.7/config":
+                client_tracer = data.get("request", {}).get("content", {}).get("client", {}).get("client_tracer", {})
+                if "extra_services" in client_tracer:
+                    extra_services = client_tracer["extra_services"]
+
+                    if extra_services is not None and len(extra_services) > 0:
+                        return True
+
+                return False
+
+        def validate(data):
+            """ Helper to validate config request content """
+            client_tracer = data["request"]["content"]["client"]["client_tracer"]
+
+            if not ("extra_services" in client_tracer):
+                raise ValidationError(
+                    "client_tracer should contain extra_services", extra_info={"client_tracer": client_tracer},
+                )
+
+            extra_services = client_tracer["extra_services"]
+
+            if extra_services is None or len(extra_services) != 1 or extra_services[0] != "extraVegetables":
+                raise ValidationError(
+                    "extra_services should be reported", extra_info={"extra_services": extra_services},
+                )
+
+            return True
+
+        interfaces.library.wait_for(remote_config_asm_extra_services_available, timeout=30)
```

Comment:
wait_for function must be in the setup. At this point, the weblog container is removed.

Reply:
fixed in 87a9c93